sdk/invoke.js

```diff
@@ -33,24 +34,21 @@ const invoke = async function invoke(accountId, funcName, apiKey, funcData) {
     logger.debug('Invoke response', { statusCode, headers, body });
     return res;
   } catch (err) {
-    let parsedError;
-    try {
-      parsedError = JSON.parse(err.error);
-    } catch (nestedErr) {
-      logger.debug('Failed to parse JSON response', { nestedErr });
-    }
-
-    if (parsedError) {
-      const errorOrMsg = parsedError.error || parsedError.message;
-      if (parsedError.stack && errorOrMsg) {
-        throw new Error(`${errorOrMsg}\n${parsedError.stack}`);
-      }
-      if (parsedError.message && parsedError.request_id) {
-        throw new Error(`${parsedError.message}\nrequest_id: ${parsedError.request_id}`);
+    if (err instanceof StatusCodeError) {
+      const requestId = err.response.headers['x-binaris-request-id'];
+      const fullError = [`RequestId: ${requestId}`];
```

Comment:
This can be undefined iiuc. Still ok imho

Reply:
I don't think it can be (as long as our services always return it).